agreement/asyncVoteVerifier.go

```diff
@@ -58,6 +61,16 @@ type AsyncVoteVerifier struct {
 	ctxCancel       context.CancelFunc
 }
 
+// RoundOffsetError is an error for when agreement far behind ledger
+type RoundOffsetError struct {
```

Comment:
Can we have this error be defined in the ledger rather than in the agreement ?
We might need to have both, and "bridge" them in the node to avoid additional dependencies.
(alternative - move it to a shared location such as data)

Reply:
So when I do this there is a circular import dependency. I'm not sure what the best solution would be here.